### PR TITLE
Clean up Readme.md

### PR DESCRIPTION
**Pull request by USTAADCOM**

### CODE REVIEW

File: apps/automation_crews/financial_crew/Readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      OR
+python ./apps/automation_crews/financial_crew/run_crew.py
+
+```
+
+### Start "Website automation" with Puppeteer
+
+```shell
+npm install
+cd ./apps/automation_crews
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/financial_crew/Readme.md:17]: correct?

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# start Only Personal Assistant Crew
+
+## Steps to Run
+**Navigate to the Project Directory:**
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:4]: where?

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Change to the directory where the `setup.sh`, `run_crew.py`, `requirements.txt` files are located. For example:
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:5]: where?

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+cd path/to/project/directory
+```
+
+### 1. Run the Setup File
+Make the setup.sh Script Executable (if necessary):
+On Linux or macOS, you might need to make the setup.sh script executable:
+```shell
+chmod +x setup.sh
+```
+Execute the setup.sh script to set up the environment, install dependencies, login to composio and
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:16]: comment not needed, its enough to say "./setup.sh" and then that file should have a comment about what it does

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+On Linux or macOS, you might need to make the setup.sh script executable:
+```shell
+chmod +x setup.sh
+```
+Execute the setup.sh script to set up the environment, install dependencies, login to composio and 
+add necessary tools:
+```shell
+./setup.sh
+```
+Now, Fill in the .env file with your secrets.
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:21]: comment not needed

File: apps/automation_crews/financial_crew/Readme.md (hunk 1 above)
Comment (USTAADCOM) [reply apps/automation_crews/financial_crew/Readme.md:17]: Resolved

File: apps/automation_crews/personal_assistant_crew/Readme.md (hunk 3 above)
Comment (USTAADCOM) [reply apps/automation_crews/personal_assistant_crew/Readme.md:5]: done

File: apps/automation_crews/financial_crew/Readme.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+### Start ONLY "Financial Crew"
+#### Requires ./src/fincancial_crew/assets/Bank-of-America-Bank-Statement-TemplateLab.com_.pdf to exist
+#### fix some imports
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/financial_crew/Readme.md:3]: is this a todo for the future?

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# start Only Personal Assistant Crew
+
+## Steps to Run
+**Navigate to the Project Directory:**
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:4]: comment not needed

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+```shell
+cd apps/automation_crews/personal_assistant_crew
+```
+Change to the directory where the `setup.sh`, `run_crew.py`, `requirements.txt` files are located. For example:
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:8]: prefer writing steps, not descriptions, e.g
setup.sh
run_crew_py
...

File: apps/automation_crews/personal_assistant_crew/Readme.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### 1. Run the Setup File
+Make the setup.sh Script Executable (if necessary):
+On Linux or macOS, you might need to make the setup.sh script executable:
```
Comment (maximiliangarmatsch) [inline apps/automation_crews/personal_assistant_crew/Readme.md:13]:
```suggestion
On Linux or macOS
```

File: apps/automation_crews/financial_crew/Readme.md (hunk 6 above)
Comment (USTAADCOM) [reply apps/automation_crews/financial_crew/Readme.md:3]: No, it was a comment added to the description. But now I have deleted it.

File: apps/automation_crews/personal_assistant_crew/Readme.md (hunk 7 above)
Comment (USTAADCOM) [reply apps/automation_crews/personal_assistant_crew/Readme.md:4]: resolve

File: apps/automation_crews/personal_assistant_crew/Readme.md (hunk 8 above)
Comment (USTAADCOM) [reply apps/automation_crews/personal_assistant_crew/Readme.md:8]: resolve

File: apps/automation_crews/personal_assistant_crew/Readme.md (hunk 9 above)
Comment (USTAADCOM) [reply apps/automation_crews/personal_assistant_crew/Readme.md:13]: resolve

File: README.md [hunk 10]
```diff
@@ -5,171 +5,27 @@
...
 - Puppeteer
+- CrewAI
+- Discord
 
 ## Use Cases
 
-- Crawl google accounts
-- Read unread emails and attached PDFs
-- Respond based on screenshot
+- Crawl google accounts
```
[the remark this answers is not in the document]
Comment (USTAADCOM) [reply README.md]: resolve

**Review by maximiliangarmatsch — APPROVED** [review]

**Pull request merged**